tests/Controllers/HistoryControllerFactoryTest/HistoryControllerFactoryExtension.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace SilverStripe\AssetAdmin\Tests\Controller\HistoryControllerFactory;
+
+use SilverStripe\Core\Extension;
+use SilverStripe\Dev\TestOnly;
+
+class HistoryControllerFactoryExtension extends Extension implements TestOnly
+{
+    public function updateIsEnabled($record)
+    {
+        return $record->Title === '2';
```

Comment:
Can you add a comment that this is referring to HistoryControllerFactoryTest.yml fixtures?